<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>808</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1172</width>
    <height>808</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1172</width>
    <height>808</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="eliminar_boton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1131</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qlineargradient(spread:pad, x1:0.542249, y1:1, x2:0.527, y2:0, stop:0 rgba(68, 0, 127, 255), stop:0.502488 rgba(170, 71, 137, 255), stop:1 rgba(255, 131, 145, 255));

border-radius: 40px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imagen_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>691</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: white;
position: relative;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selecc_boton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>580</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#selecc_boton:pressed{
	padding-left: 5px;
	padding-top:5px;
	background-color: rgba(255, 131, 145, 255);
}

QPushButton#selecc_boton:hover{
	background-color:#61A749;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Seleccionar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eliminar_boton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>580</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#eliminar_boton:pressed{
	padding-left: 5px;
	padding-top:5px;
	background-color: rgba(255, 131, 145, 255);
}

QPushButton#eliminar_boton:hover{
	background-color:#D85D49;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guardar_boton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>630</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#guardar_boton:pressed{
	padding-left: 5px;
	padding-top:5px;
	background-color: rgba(255, 131, 145, 255);
}

QPushButton#guardar_boton:hover{
	background-color:#33AAC7;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salir_boton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton#salir_boton:pressed{
	background-color: red;
}

QPushButton#salir_boton{
	border-radius: 15px;
background-color: white;
}


</string>
    </property>
    <property name="text">
     <string>❌​</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F3D52F;
color: white;

border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;PIMP&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ByN_boton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#ByN_boton{
background-color: #EAC600;
color: white;
border-radius: 15px;
}


QPushButton#ByN_boton:hover{
	background-color: black;
	color:white;
}
</string>
    </property>
    <property name="text">
     <string>Blanco y Negro</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;FILTROS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="negativo_boton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#negativo_boton{
background-color: #EAC600;
color: white;
border-radius: 15px;
}


QPushButton#negativo_boton:hover{
	background-color: #9736D4;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Negativo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sepia_boton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#sepia_boton{
background-color: #EAC600;
color: white;
border-radius: 15px;
}


QPushButton#sepia_boton:hover{
	background-color: #C8B88E;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Sepia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blur_boton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#blur_boton{
background-color: #EAC600;
color: white;
border-radius: 15px;

}


QPushButton#blur_boton:hover{
	-webkit-backdrop-filter:blur(15px);
	backdrop-filter: blur(15px);
	background-color: rgba(243,243,243,0.2);
	color:white;


}</string>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="circulo_boton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#circulo_boton{
background-color: #EAC600;
color: white;
border-radius: 15px;

}


QPushButton#circulo_boton:hover{
	-webkit-backdrop-filter:blur(15px);
	backdrop-filter: blur(15px);
	background-color: #45E6BF;
	color:white;


}</string>
    </property>
    <property name="text">
     <string>Circulo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eliminar_filtros_boton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#eliminar_filtros_boton{
background-color: #EAC600;
color: white;
border-radius: 15px;

}


QPushButton#eliminar_filtros_boton:hover{
	-webkit-backdrop-filter:blur(15px);
	backdrop-filter: blur(15px);
	background-color: #ffffff;
	color:white;


}</string>
    </property>
    <property name="text">
     <string>❌​</string>
    </property>
   </widget>
   <widget class="QLabel" name="histograma_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: white;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;HISTOGRAMA&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Rotarm90_boton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>570</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Rotarm90_boton{
background-color: #EAC600;
color: white;
border-radius: 20px;
}


QPushButton#Rotarm90_boton:hover{
		-webkit-backdrop-filter:blur(15px);
	backdrop-filter: blur(15px);
	background-color: rgba(243,243,243,0.2);
	color:white;
}</string>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eliminar_ruido_boton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#eliminar_ruido_boton{
background-color: #EAC600;
color: white;
border-radius: 15px;

}


QPushButton#eliminar_ruido_boton:hover{
	-webkit-backdrop-filter:blur(15px);
	backdrop-filter: blur(15px);
	background-color: #E6459D;
	color:white;


}</string>
    </property>
    <property name="text">
     <string>Ruido</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mejorar_boton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura Maxi CG Bold</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#mejorar_boton{
background-color: #EAC600;
color: white;
border-radius: 15px;

}


QPushButton#mejorar_boton:hover{
	-webkit-backdrop-filter:blur(15px);
	backdrop-filter: blur(15px);
	background-color: #FFEE82;
	color:white;


}</string>
    </property>
    <property name="text">
     <string>Trazado</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
